python/lsst/analysis/drp/colorColorFitPlot.py

```diff
@@ -385,15 +484,18 @@ def colorColorFitPlot(self, catPlot, plotInfo, fitParams):
         minXs = -1*np.min(np.fabs(percsDists))
         maxXs = np.min(np.fabs(percsDists))
         plotPoints = ((dists < maxXs) & (dists > minXs))
-        xs = np.array(dists)[plotPoints]
+        xs = np.array(dists)[plotPoints]*statsUnitScale
```

Comment:
Probably the units should be accounted for in the distance calculation function.

Reply:
Oh, sure.  I won't change the `perpDistance` function, but I'll do the scaling above where `dists` is computed.